go.mod

```diff
@@ -6,27 +6,28 @@ require (
 	github.com/blang/semver/v4 v4.0.0
```

Comment:
The fetch command to install goreleaser is broken and has no relation with this PR.